authors/apps/authentication/tests/test_verify_user.py

```diff
@@ -18,9 +18,27 @@ def test_unsuccessfull_verification(self):
         self.signup_user('Elchapo', 'el@chapo.com', 'Maina9176')
         self.verify_user('el@chapo.com')
         token = self.generate_jwt_token('el@chapo.com', 'Elchapo')
-        url = "/api/users/activate/" + token
+        url = self.verification_of_user + token
 
         response = self.client.get(url)
         
         self.assertEqual(response.data['message'], "Your email is already verified")
-        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
\ No newline at end of file
+        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
+
+    def test_ubsuccessfull_verification_token(self):
```

Comment:
Hello @Bakley ,
Would you correct the typo in the method i.e `unsuccessful` instead of `ubsuccessful`. 
 In addition, I think this method tests invalid_verification_token not an unsuccessful token

Reply:
Okay, working on fixing that